src/components/structures/AutoHideScrollbar.tsx

```diff
@@ -52,14 +52,18 @@ export default class AutoHideScrollbar extends React.Component<IProps> {
     }
 
     public render() {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const { className, onScroll, onWheel, style, tabIndex, wrappedRef, children, ...otherProps } = this.props;
+
         return (<div
+            {...otherProps}
```

Comment:
spaces

Reply:
Hmm, we don't have a single instance of such in react-sdk, where is this documented? (two here are in comments)

![image](https://user-images.githubusercontent.com/2403652/122174918-15faf680-ce7b-11eb-9a16-13943bdaa7cf.png)

Reply:
we have several instances where react's `{}` gets padded by spaces, and our latest object linter rules seem to want this.

Reply:
We actively avoid spaces in react props though, in other JSX we do space out

Reply:
ah, true.